packages/check-iam-wildcards/checkAsterisks.ts

```diff
@@ -0,0 +1,61 @@
+import { iam } from './security.serverless';
+import { IamRole } from 'serverless/aws';
```

Comment:
**issue |** there is no `serverless/aws` as devDep in package-lock

Reply:
script copies to project root. dir